.github/workflows/e2e-master.yml

```diff
@@ -137,11 +137,17 @@ jobs:
       env:
         TERM: xterm
 
-    - name: Upload Cypress recording upon failure
+    - name: Generate Mochawesome Reports
+      run: |
+        yarn generate:cypress:html:report
```

Comment:
Nit: Why do we have multi-line syntax for a single command?

Reply:
Nothing in special... will add in oneline

Reply:
https://github.com/metabase/metabase/pull/21727/commits/decfeb73628f6f75dd1bf731962d9e3a5fabe269

Don't forget to fix this in **both** places.